### DISCUSSION TITLE
bpozdena's OneDriveGui 1.1.1 does not see onedrive 2.5.3 but's it's syncing in the terminal

### DISCUSSION THREAD

**Discussion by Pauly-Walnuts62** [bpozdena/OneDriveGUI#212] [General]
I've installed Onedrive v.2.5.3 by abraunegg, and when I run it from the terminal, it's "doing it's thing". 

Starting a sync with Microsoft OneDrive
Syncing changes from Microsoft OneDrive ...
Sync with Microsoft OneDrive is complete

This after it downloaded some 45 GB's. 

However, while it's running, and I open OneDriveGui 1.1.1, I see a message saying that Business Shared Folder has been deprecated. I'm nobody's business. Just a short little old man playing with Linux Mint to keep the old noodles from disintegrating. 

My account is not "business" to my knowledge. I do have an upgraded account in that I have 1 TB storage, and Office 365... 

What IS a feller to do?

**Comment by bpozdena** [comment]
From [readme](https://github.com/bpozdena/OneDriveGUI/blob/main/README.md) file: 


> [!IMPORTANT]
> If you are using `sync_business_shared_folders` in of OneDrive v2.4.0, you will need to manually remove it from the config file. Check the [client documentation](https://github.com/abraunegg/onedrive/blob/onedrive-v2.5.0-release-candidate-1/docs/business-shared-items.md) for more details.

**Reply by Pauly-Walnuts62** [reply]
Like I say. I'm running v.2.53, not 2.40.

And I did remove that, I rebooted the machine after I set it all up and edited the config file, and no joy. However, due to another project, I rebooted again, and I no longer get the error about the share, but now it just says onedrive sync is starting. I get a green dot in the upper right corner, which turns red only a moment later. I did deliberately copy and paste a folder of photos to give it something to do, pressed the arrow key to start it, but the red dot appeared again a moment later.

Thanks!
